src/worker/views/teamStatDists.ts

```diff
@@ -1,6 +1,8 @@
 import { idb } from "../db";
 import { g, helpers } from "../util";
 import type { UpdateEvents, ViewInput } from "../../common/types";
+import { TEAM_STATS_DIST_TABLE } from "../../common";
```

Comment:
Since this is the only place importing it, and that probably will not change, you should just inline this below rather than put it in common.

Even better might be to make this use TEAM_STATS_TABLES with a dropdown to select the table, just like you did in PlayerStatDists. But don't feel obligated to do that... even with just a table of team stats, it's better than nothing!